http/write_handler.go

```diff
@@ -134,7 +134,7 @@ func (h *WriteHandler) handleWrite(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	logger := h.Logger.With(zap.String("org", req.Org), zap.String("bucket", req.Bucket))
+	logger := h.log.With(zap.String("org", req.Org), zap.String("bucket", req.Bucket))
```

Comment:
nit: this is just a refactor so it's not a big deal, but for context-sensitive tasks like this a [logger.Operation](https://github.com/influxdata/influxdb/blob/master/logger/fields.go#L106) is probably a good choice.